ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -1068,3 +1068,35 @@ def test_tensorflow_zero_fraction(
         fn_tree="math.zero_fraction",
         value=x[0],
     )
+
+
+# argmin
+@handle_cmd_line_args
+@given(
+    dtype_and_x=statistical_dtype_values(function="argmin"),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.argmin"
+    ),
+)
+def test_tensorflow_argmin(
+    dtype_and_x,
+    as_variable,
+    num_positional_args,
+    native_array,
+    fw,
+):
+    input_dtype, x, axis = dtype_and_x
+    if isinstance(axis, tuple):
+        axis = axis[0]
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        frontend="tensorflow",
+        fn_tree="math.argmin",
+        input=x[0],
+        axis=axis,
+        output_type="int64",
```

Comment:
Instead of simply passing a set value `int64`, sample values for `output_type` should be generated in `@given` and passed to the `test_frontend_function` as you have done for the rest of the input arguments.

Reply:
If you're not sure how to do this, you can have a look at how dtypes are drawn in test_torch_full here https://github.com/unifyai/ivy/blob/1dc270401a1d9bd992de559c816d707149e61884/ivy_tests/test_ivy/test_frontends/test_torch/test_creation_ops.py#L12.